File: web3rpc/rpc-specs/code-samples/javascript/eth/block/getHeaderByNumber.js
const { JsonRpcProvider } = require("@klaytn/ethers-ext");
(() => {
const provider = new JsonRpcProvider("https://public-en-baobab.klaytn.net");

    const blockNumber = 1
    
    provider.eth.getHeaderByNumber(blockNumber, {}, (err, data, response) => {
        console.log(data);
    });
}
)()

File: web3rpc/rpc-specs/code-samples/python/eth/block/getHeaderByNumber.py
from web3 import Web3
from web3py_ext import extend

host = "https://api.baobab.klaytn.net:8651"

blockTag = "0x1b4"

w3 = Web3(Web3.HTTPProvider(host))
eth_response = w3.eth.get_header_by_number(blockTag)

print(eth_response)
